client/src/store/panelStore.js

```diff
@@ -22,13 +22,20 @@ const getters = {
 };
 
 const actions = {
-    initCurrentPanelView: async ({ commit, state }, siteDefaultPanelView) => {
-        const panelView = state.currentPanelView || siteDefaultPanelView;
+    initCurrentPanelView: async ({ commit, state, dispatch }, siteDefaultPanelView) => {
+        let panelView = state.currentPanelView || siteDefaultPanelView;
```

Comment:
```suggestion
        const panelView = state.currentPanelView || siteDefaultPanelView;
```

Reply:
(never reassigned, unless you had other outstanding changes coming?)

Reply:
Nope - I had switched this around 15 different ways trying to figure out how to get this to work and left it a mess. Thanks for the catch!